test/libsolidity/InlineAssembly.cpp

```diff
@@ -291,27 +320,27 @@ BOOST_AUTO_TEST_CASE(oversize_string_literals)
 
 BOOST_AUTO_TEST_CASE(assignment_after_tag)
 {
-	BOOST_CHECK(successParse("{ let x := 1 { tag: =: x } }"));
+	BOOST_CHECK(successParse("{ let x := 1 { 7 tag: =: x } }"));
```

Comment:
I think you got that right, yes. There can of course be other stack elements between `x` and `something`. And `{ 2 { let x := 1 } pop }` is valid. Whether stack slots are named or not does not matter.